sesje/2015-02-11/remiander.py

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
do skasowania, służył do sprawdzenia poprawności. Chciałem załączyć fragmenty tego pliku w `remainder.md` ale nie znalazłem w dokumentacji jak to zrobić.